app/models/fee_scheme.rb

```diff
@@ -15,6 +15,7 @@ class FeeScheme < ApplicationRecord
   scope :ten, -> { where(version: FeeScheme::TEN) }
   scope :eleven, -> { where(version: FeeScheme::ELEVEN) }
   scope :twelve, -> { where(version: FeeScheme::TWELVE) }
+  scope :lgfs_ten, -> { lgfs.where(version: FeeScheme::TEN) }
```

Comment:
Do we need a scope to differentiate between schemes 9 & 10 for AGFS and LGFS?

Reply:
Good question. The `lgfs_ten` scope I added isn't being used anywhere. I thought we might need it but we don't - 
I just forgot to remove it. I'll delete it from the commit.

Where the numbered scopes are used, they're combined with the `agfs` and `lgfs` scopes, so we do `FeeScheme.lgfs.nine` or `FeeScheme.agfs.nine` or `FeeScheme.lgfs.ten` or `FeeScheme.agfs.ten`.